libs/adapters/tsconfig.build.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build"
```

Comment:
no strict when building?

Reply:
I believe the `strict` property will be inherited from the base tsconfig.